resources/external_control.urscript

```diff
@@ -98,8 +98,8 @@ socket_open("{{SERVER_IP_REPLACE}}", {{SERVER_PORT_REPLACE}}, "reverse_socket")
 control_mode = MODE_UNINITIALIZED
 thread_move = 0
 global keepalive = -2
+params_mult = socket_read_binary_integer(1+6+1, "reverse_socket", 0)
```

Comment:
Technically not really part of this merge, is it? Though I don't really mind sneaking it in here.

Reply:
Yes, I took the script from the ROS application, not the current version here.